docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,2005 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        Returns
</pre><p>Please update the description of the return values</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Right! I missed it! will be fixed in next set of commits.</p>